### DISCUSSION TITLE
Consider adding Highlights badges as well

### DISCUSSION THREAD

**Discussion by thomscoder** [Schweinepriester/github-profile-achievements#34] [Ideas]
I would like to receive your feedback in adding [Highlights badges](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile:~:text=When%20you%20participate%20in%20certain%20programs%2C%20GitHub%20automatically%20displays%20a%20badge%20on%20your%20profile.) as well. 
One is unlockable via the galaxy brain triage. The other are independent. I think they complete the README.md

**Comment by kirollosatef** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Reply by thomscoder** [reply]
Hi @kirollosatef 
Thanks for the reply. Yes I'm aware of the existence of that repo (i contributed to that :) ). I would like to add the highlights badges as well, so I'm waiting for the owner opinion on this.
Thanks again

**Comment by Schweinepriester** [comment]
Mh, I feel like currently the [official docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile) document them best possible (I even [contributed to them recently](https://github.com/github/docs/pull/17870) B-)), so I don't feel like just copying.

Having said that, a section mentioning them and linking there seems sensible for people looking for them.
Also pulling the data (icons & texts) for future archival purposes or if they remove those docs as well seems useful (even though unless they pull https://github.com/github/docs as a whole it will be in the history ~forever).
Just to be sure I put [a new snapshot](https://web.archive.org/web/20220620175927/https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) in the archive.

**Reply by Schweinepriester** [reply]
here we go: https://github.com/Schweinepriester/github-profile-achievements#highlights

**Reply by thomscoder** [reply]
Fast af 😂 thank you for your feedback Kai!
